creator-node/test/snapbackSM.test.js

```diff
@@ -646,21 +646,9 @@ describe('test SnapbackSM', function () {
   it('[aggregateReconfigAndPotentialSyncOps] if the self node is the secondary and a primary spId is different from what is on chain, issue reconfig', async function () {
     const snapback = new SnapbackSM(nodeConfig, getLibsMock())
 
-    // Mock these as very nodes that completed only successful syncs
```

Comment:
oh ok lol, i guess removing this entirely also works since it will just return undefined which happens to pass for the wrong reason

Reply:
yeah, it's not used in the flow so we shouldn't mock it

Reply:
ahhh i see, you only removed it from tests where self is secondary, makes sense 👍